vm/actor/tests/account_actor_test.rs

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+mod common;
+
+use actor::{account::State, ACCOUNT_ACTOR_CODE_ID, SYSTEM_ACTOR_ADDR, SYSTEM_ACTOR_CODE_ID};
+use address::Address;
+use common::*;
+use db::MemoryDB;
+use vm::{ExitCode, Serialized};
+
+macro_rules! account_tests {
+    ($($name:ident: $value:expr,)*) => {
+        $(
+            #[test]
+            fn $name() {
+                let (addr, exit_code) = $value;
+
+                let bs = MemoryDB::default();
+                let receiver = Address::new_id(100).unwrap();
+                let mut rt = MockRuntime::new(&bs, receiver.clone());
+                rt.caller = SYSTEM_ACTOR_ADDR.clone();
+                rt.caller_type = SYSTEM_ACTOR_CODE_ID.clone();
+                rt.expect_validate_caller_addr(&vec![SYSTEM_ACTOR_ADDR.clone()]);
+
+                if exit_code.is_success() {
+                    let _ = rt
```

Comment:
Suggestion: remove `let _`

Reply:
Was sure clippy would complain, but it didnt complain locally...